core/server/data/migration/reset.js

```diff
@@ -12,11 +11,23 @@ var Promise      = require('bluebird'),
  * Deletes all the tables defined in the schema
  * Uses reverse order, which ensures that foreign keys are removed before the parent table
  *
+ * @TODO:
+ * - move to sephiroth
+ * - then deleting migrations table will make sense
```

Comment:
the idea here is: 
- sephiroth will take over reset tables (`sephirote reset` or `sephiroth.commands.reset`)
- it will offer a hook to Ghost
- so maybe we will create a file `/data/migrations/reset.js`, which is an exposed function to delete all our custom tables and what ever we want